sprints/dashboard/models.py

```diff
@@ -124,12 +124,36 @@ def assignee_time(self) -> int:
 
         return max(self.time_estimate - self.review_time, 0)  # We don't want negative values here.
 
+    def get_review_time_from_story_points(self):
+        issue_story_points = self.story_points
+        review_hours_by_story_points = settings.SPRINT_HOURS_RESERVED_FOR_REVIEW
+        if issue_story_points is None:
+            return review_hours_by_story_points.get("null")
+
+        # Default is the null value, in case no numeric value has been defined on the env variable
+        nearest_value = "null"
+        minimum_value_difference = None
+        # Iterate over the defined values, and look for the nearest one
+        for story_points in review_hours_by_story_points:
+            if story_points == "null":
+                continue
+            story_point_num_value = float(story_points)
+            value_difference = abs(story_point_num_value - issue_story_points)
+            if nearest_value == "null" or \
+               (value_difference == minimum_value_difference and story_point_num_value > float(nearest_value)) or \
+               value_difference <= minimum_value_difference:
+                minimum_value_difference = value_difference
+                nearest_value = story_points
+
+        return review_hours_by_story_points.get(nearest_value)
```

Comment:
This looks quite complex. Why don't we just do something like this?
```python
# Some explanation that the value is not defined, so we're getting the closest one - this should also go to the docstring of this function.
defined_story_points = sorted([key for key in review_hours_by_story_points.keys() if key is not None])
nearest_story_points min(defined_story_points, key=lambda x:abs(x - self.story_points))
return review_hours_by_story_points[nearest_story_points]
```

Reply:
Sure, that looks way simple